[TW review] Merchant Products feature (#161)
spryker/spryker.github.io: docs/marketplace/user/features/202106.0/merchant-products.md

> Merchant Products are the products that a Merchant owns. Besides creating the offers for the products that the Marketplace Administrator suggests, a Merchant can create their own unique products. These products possess the same characteristics the usual abstract and concrete products have, but in addition, every such product has Merchant-related information such as merchant reference. Merchants manage stock, set prices 

Reviewer: offers for the products - of other Marchants - not only Marketplace Administrator

Author: will add

## Unique and shared merchant products

> Merchant products can be unique or shared between several merchants. In case the product is shared, it has the owner merchant and other merchants are able to create offers for this product in the Merchant Portal.

Reviewer: shared between several merchants can be misleading, it's more about allowing to create offers or not (think of operator /main merchant case that only allows offers in their marketplace)

Author: good point, thanks, will update the documentation

Currently, you can define whether the product is shared or not via [data importer](https://spryker-docs.herokuapp.com/docs/marketplace/dev/data-import/202106.0/file-details-merchant-product-csv.html). 

## Merchant products in the Storefront

Merchant products are displayed in the Storefront when the following conditions are met:

1. The merchant who owns the product is [*Active*](https://spryker-docs.herokuapp.com/docs/marketplace/user/back-office-user-guides/202106.0/marketplace/merchants/managing-merchants.html#activating-and-deactivating-merchants).

> 2. The product visibility state is `Online`.

Reviewer: the product Status state is `Active`

Author: ✓

3. The product is defined for the current store.

> 4. The product has stock.

Reviewer: or it can be always in stock. also out of stock will still be on yves but with the option to sign for back in stock. and name and tax is also mandatory and validity dates are in the present

### Merchant product on the product details page

> Merchant product appears first in the *Sold by* list together with the product offers from other merchants. For a Buyer, it doesn't matter what they are buying a product offer or a merchant product, however in the system, different entities are defined.

Reviewer: appears first **on** the list and even better top of the list

Author: ✓

> Merchant product appears first in the *Sold by* list together with the product offers from other merchants. For a Buyer, it doesn't matter what they are buying a product offer or a merchant product, however in the system, different entities are defined.

Reviewer: it doesn't matter whether they are

Author: ✓

![Merchant product on PDP](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/merchant-product-on-pdp.png)

### Merchant products during the Checkout

> Merchant products are grouped by the merchant and get split into the shipments by merchant.

Reviewer: maybe let's not get into checkout stuff here (as a buyer can split to more shipments etc and the sold by is true to wishlists and in the future shopping list etc), and just say the merchant product will have sold by in these and these pages

Author: will add the succession of pages where the *Sold by* information is displayed.

## Merchant Products in the Back Office

> A Marketplace Administrator can filter the products belonging to merchants in the Back Office.

Reviewer: and 
1. edit the products if needed
2. create products as main merchants 
3. what about mentioning Merchant portal and that merchant can create/edit merchant products there

Author: 3 - There is a section "Merchant products in the Merchant Portal" down on this page where all these actions are described.

---

> title: Merchant Products

Reviewer 2: The feature is called "Marketplace Products", so please change the title and links.

description: This document contains concept information for the Merchant Products feature in the Spryker Commerce OS.
template: concept-topic-template
---

Merchant Products are the products that a Merchant owns. Besides creating the offers for the products of other Merchants or the ones that the Marketplace Administrator suggests, a Merchant can also create their own unique products. These products possess the same characteristics the usual abstract and concrete products have, but in addition, every such product has Merchant-related information such as merchant reference. Merchants manage stock and set prices in the Merchant Portal. 

> Merchants can allow creating offers for their unique products for other merchants. It is defined with the help of `is_shared` parameter of the [data importer](https://spryker-docs.herokuapp.com/docs/marketplace/dev/data-import/202106.0/file-details-merchant-product-csv.html). 

Reviewer 2: ```suggestion
Merchants can allow other merchants to create offers for their unique products. This possibility is defined with the help of `is_shared` parameter of the [merchant product data importer]/docs/marketplace/dev/data-import/{{ page.version }}/file-details-merchant-product-csv.html). 
```

> ## Merchant products in the Storefront

Reviewer 2: ```suggestion
## Merchant products on the Storefront
```

> Merchant products are displayed in the Storefront when the following conditions are met:

Reviewer 2: ```suggestion
Merchant products are displayed on the Storefront when the following conditions are met:
```

1. The product status is *Active*.

> 2. The merchant who owns the product is [*Active*](https://spryker-docs.herokuapp.com/docs/marketplace/user/back-office-user-guides/202106.0/marketplace/merchants/managing-merchants.html#activating-and-deactivating-merchants).

Reviewer 2: ```suggestion
2. The merchant who owns the product is [*Active*](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/marketplace/merchants/managing-merchants.html#activating-and-deactivating-merchants).
```

3. The product visibility state is `Online`.
4. The product is defined for the current store.
5. The product has stock or is always in stock.
6. The current day is within the range of the product validity dates.

### Merchant product on the product details page

> Merchant product appears on top of the the *Sold by* list together with the product offers from other merchants. For a Buyer, it doesn't matter whether they are buying a product offer or a merchant product, however in the system, different entities are defined.

Reviewer 2: ```suggestion
Merchant product appears on top of the the *Sold by* list together with the product offers from other merchants. For a buyer, it doesn't matter whether they are buying a product offer or a merchant product, however in the system, different entities are defined.
```

![Merchant product on PDP](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/merchant-product-on-pdp.png)

The merchant product is also displayed with the *Sold By* field defining the merchant on the following pages:

- Cart page
- Wishlist
- Shipment page of the Checkout
- Summary page of the Checkout
- Order Details of the customer account

![Merchant product on PDP](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/add-merchant-product-to-wl-and-from-wh-to-cart.gif)

### Searching and filtering merchant products
When the merchant name is entered in the catalog search, not only the offers but also the products belonging to this merchant are displayed. By selecting a merchant name in the filter, products from this merchant are also displayed.

![Search for merchant products](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/search-for-products-by-name-and-sku.gif)

## Merchant products in the Back Office

> Before new merchant products can be seen by the Buyers in the Storefront, they must be activated <!---LINK--> in the Back Office.

Reviewer 2: ```suggestion
Before new merchant products become visible on the Storefront, they must be activated <!---LINK--> in the Back Office.
```

A Marketplace Administrator can filter the products belonging to certain merchants in the Back Office.

![merchants-switcher-on-products](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+product/filter-merchant-productsby-merchant-back-office.gif)

Also, Marketplace administrators can edit products, if needed, and create products when acting as the main merchant<!---LINK-->.

## Merchant products in the Merchant Portal

> Merchants create and manage their products <!---LINK--> in the Merchant Portal. They can define their own prices, stock, attributes etc. for their products. See Managing merchant products <!---LINK--> for detailed instructions on how to do that.

Reviewer 2: ```suggestion
Merchants create and manage their products <!---LINK--> in the Merchant Portal. They can define prices, stock, attributes etc. for their products. See Managing merchant products <!---LINK--> for details on how to do that.
```

> ## Merchant products and API

Reviewer 2: This section can be moved to the Marketplace Products feature walkthrough for devs.

Spryker Marketplace provides API to:

- [Retrieve abstract products](/docs/marketplace/dev/glue-api-guides/{{ page.version }}/abstract-products/retrieving-abstract-products.html)
- [Retrieve concrete products](/docs/marketplace/dev/glue-api-guides/{{ page.version }}/concrete-products/retrieving-concrete-products.html)
- [Retrieve abstract product lists](/docs/marketplace/dev/glue-api-guides/{{ page.version }}/abstract-products/retrieving-abstract-product-lists.html)

## Related Business User articles

| MERCHANT PORTAL USER GUIDES  | BACK OFFICE USER GUIDES |
| -------------------- | ----------------------- |
| Managing merchant products <!---LINK--> | [Editing abstract products](https://documentation.spryker.com/docs/editing-abstract-products) |
| | [Editing a product variant](https://documentation.spryker.com/docs/editing-a-product-variant) |

{% info_block warningBox "Developer guides" %}

> Are you a developer? See Merchant Products <!---LINK--> feature walkthrough for developers.

Reviewer 2: ```suggestion
Are you a developer? See Marketplace Products <!---LINK--> feature walkthrough for developers.
```
